network/cmd/cmd.go

```diff
@@ -151,7 +179,11 @@ func listCommand() *cobra.Command {
 			return nil
 		},
 	}
+
 	cmd.Flags().StringVar(&sortFlag, "sort", sortFlagTime, "sort the results on either time or type")
+	cmd.Flags().StringVar(&rangeStart, "start", "", "inclusive start of lamport clock range")
```

Comment:
isn't there an `IntVar`?

Reply:
Yes but I don't think the behavior for an unspecified value is very nice. StringVar is nice in that you get an empty string if the option was not specified.

I'll take a second look and make sure IntVar  doesn't work for our use case.